pom.xml

```diff
@@ -1998,6 +1998,8 @@
                                 flag may be dropped after resolution of
                                 https://bugs.openjdk.java.net/browse/JDK-8155674. -->
                                 <arg>-XDcompilePolicy=simple</arg>
+                                <!-- XXX: Drop! This is a test. -->
+                                <arg>--should-stop=ifError=INIT</arg>
```

Comment:
This is me testing the changes for google/error-prone#4595. The build passes both without this flag and with `INIT`, which based on the release notes shouldn't happen. (When I replace `INIT` with a bogus value the compiler does complain, so the flag is not ignored.)

It appears that the issue is that flag validation happens [here](https://github.com/google/error-prone/blob/82a216870b043e841cad14eddb4568b8edcb9eff/check_api/src/main/java/com/google/errorprone/BaseErrorProneJavaCompiler.java#L72-L74), while `ErrorProneJavacPlugin` [bypasses](https://github.com/google/error-prone/blob/82a216870b043e841cad14eddb4568b8edcb9eff/core/src/main/java/com/google/errorprone/ErrorProneJavacPlugin.java#L34-L35) that logic by directly invoking [this method](https://github.com/google/error-prone/blob/82a216870b043e841cad14eddb4568b8edcb9eff/check_api/src/main/java/com/google/errorprone/BaseErrorProneJavaCompiler.java#L83-L96).

Reply:
^ Follow-up [here](https://github.com/google/error-prone/issues/4595#issuecomment-2423713257).